### ISSUE TITLE
With focus on Layers control icon, NVDA announces "Layers link"

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#529]
I believe this might not be correct.  @Malvoz ?  When you hit "Enter" it's not activating a link, but displaying what amounts to a fancy menu (the layers list, with associated controls and settings). Should this be announced as something more meaningful?  Dialog, perhaps?

**Comment by Malvoz** [comment]
Yes the layer control is essentially a button that opens a menu, and should not be conveyed/behave as a link.

This issue is already documented in https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/247#layer-control:

> ## Layer control

> - [ ] Should have `role="button"`.

I think we can close this issue out in favor of https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/19.
